src/registrar/management/commands/delete_expired_domains_not_setup.py

```diff
@@ -0,0 +1,92 @@
+from django.core.management import BaseCommand
+from registrar.models import Domain, UserDomainRole
+import logging
+import argparse
+from django.utils import timezone
+from datetime import timedelta
+from registrar.utility.email import EmailSendingError, send_templated_email
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """Domains that, (1) have DNS status "Unknown" or "DNS Needed" and (2) are 7+ days past their expiration date,
+    are marked "DELETED" in the registrar and deleted in the registry."""
+
+    def handle(self, *args, **options):
+        domains_to_be_deleted = self.get_domains()
+        dry_run = options.get("dry_run", False)
+
+        if not dry_run:
+            deleted_domains = self.delete_domains_and_send_notif_emails(domains_to_be_deleted)
+            self.logging_message(dry_run, deleted_domains)
+        else:
+            self.logging_message(dry_run, domains_to_be_deleted)
+
+    def logging_message(self, dry_run, domains):
+        count = len(domains)
+        if dry_run:
+            logger.info(f"DRY RUN MODE - No changes will be made\n {count} domains will be deleted")
+        else:
+            if count > 1:
+                logger.info(f"{count} domains have been deleted.")
+            else:
+                text = "No" if count == 0 else "1"
+                logger.info(f"{text} domain has been deleted.")
```

Comment:
Since we're letting the user know which domains have been deleted at the end of the script, I think it'd be ok to split this out to the 0 domains and 1 domain case and let the user know which 1 domain got deleted!

```suggestion
elif count == 1:
                logger.info(f"1 domain was deleted: {domains}.")
else:
                logger.info(f"No domains were deleted.")
```